ora2/jobs/AddCourseToSandbox.groovy

```diff
@@ -0,0 +1,56 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    * folder-name: some-folder
+    * credential : your-ssh-credential-name
+    * email : e-mail-contact@example.com
+*/
+
+package ora2.jobs
+
+import org.yaml.snakeyaml.Yaml
+import org.yaml.snakeyaml.error.YAMLException
+
+class AddCourseToSandbox {
+
+    public static def job = { dslFactory, extraVars ->
+        return dslFactory.job(extraVars.get('folder-name', 'ora2') + '/add-course-to-sandbox') {
+                keepDependencies(false)
+                properties {
+                    rebuild {
+                        autoRebuild(false)
+                    }
+                    parameters {
+                        stringParam(
+                            'SANDBOX_BASE',
+                            'ora2',
+                            "For example, when the sandbox is foo.sandbox.edx.org, the sandbox_base is 'foo'"
+                        )
+
+                    }
+                    throttleJobProperty {
```

Comment:
I think these are all of the defaults, I'll try running without them enabled to see what XML gets generated.